notebooks/spec2vec/tutorial.ipynb

```diff
@@ -256,15 +256,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 8,
    "metadata": {},
    "outputs": [],
    "source": [
-    "writer = pd.ExcelWriter(f\"spec2vec_spectra_matches.xlsx\", engine='xlsxwriter')\n",
-    "\n",
-    "for i, matches in enumerate(result):\n",
-    "    matches.to_excel(writer, sheet_name=f'spectrum #{i+1}')\n",
-    "writer.save()"
+    "import os\n",
+    "os.makedirs(\"matches\", exist_ok=True)\n",
+    "for i, spectrum_matches in enumerate(spectra_matches):\n",
+    "    spectrum_matches.to_csv(f\"matches/spectrum_{i}.csv\")"
```

Comment:
So here we are saving each dataframe in its own csv file in a "matches" folder. Sounds good to me. We should just update the markdown description of the cell to

> For readability, each dataframe is saved in its own csv file.